content/python/concepts/strings/terms/join/join.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.join()'
+Description: 'Concatenates all items from an iterable into a single string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.join()` method concatenates all items from an iterable into a single string.
+
+## Syntax
+
```

Comment:
```suggestion

The `.join()` method is called on by being appended to a `separator` string:

```

Reply:
@Dusch4593 I like this sentence but I feel like "appended" here is forced. Can we just write it as, "The `.join()` method is called on a `separator` string:" and find another place for "append".

Reply:
@Dusch4593 I like this sentence but I feel like "appended" here is forced. Can we just write it as, "The `.join()` method is called on a `separator` string:" and find another place for "append".